driver/src/main/com/mongodb/DBCollectionChangeStreamOptions.java

```diff
@@ -17,7 +17,7 @@
 package com.mongodb;
 
 import com.mongodb.client.model.Collation;
-import com.mongodb.client.model.FullDocument;
+import com.mongodb.client.model.changestream.FullDocument;
```

Comment:
Nowhere else to comment on this but here...

Should DBCollection.watch methods be changes to return 

   Iterator<ChangeStreamOutput<DBObject>>

which in itself is a bit weird for DBCollection, but at least safer for the way it would treat the resume token.  

Alternatively, consider just removing change stream support from DBCollection entirely, given how awkward it is.